src/lib/y2storage/boot_requirements_strategies/base.rb

```diff
@@ -95,6 +95,11 @@ def errors
           res << SetupError.new(message: error_message)
         end
 
+        if too_small_boot?
+          error_message = _("Device mounted at '/boot' is too small to contain kernel.")
```

Comment:
Missing determinants "**The** device mounted at '/boot' is too small to contain **a** kernel"